features/step_definitions/common_steps.rb

```diff
@@ -312,3 +312,7 @@
     loop until page.evaluate_script('jQuery.active').zero?
   end
 end
+
+And /^I wait (\d+) seconds/ do |seconds|
```

Comment:
It might be nice to add an optional inline comment to this step so that you can give some explanation about why, i.e. `And I wait 2 seconds to ensure the reset token is valid`

Reply:
Nice. Have added.